flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/OverWindowTestPrograms.java

```diff
@@ -227,141 +227,63 @@ public class OverWindowTestPrograms {
                                                             3L,
                                                             300L,
-                                                            4L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            4L,
-                                                            400L,
-                                                            5L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            4L,
-                                                            400L,
-                                                            8L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            5L,
-                                                            500L,
-                                                            10L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            5L,
-                                                            500L,
-                                                            13L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            6L,
-                                                            600L,
-                                                            16L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            6L,
-                                                            600L,
                                                             19L))
                                             .build())
+                            .runSql(
+                                    "INSERT INTO sink_t SELECT key, val, ts, SUM(val) OVER ("
+                                            + "PARTITION BY key "
```

Comment:
```suggestion
                                            + "PARTITION BY val "
```
I would not use the same sort key as partition key because then each partition just has a single sort key.
IMO it would be better to swap the partition key and sort key (compared to the other tests).